plugin/src/test/java/io/jenkins/plugins/casc/SampleBenchmark.java

```diff
@@ -15,7 +15,13 @@
         @Nonnull
         @Override
         protected String getResourcePath() {
-            return "io/jenkins/plugins/casc/benchmarks.yml";
+            return "benchmarks.yml";
+        }
+
+        @Nonnull
+        @Override
+        protected Class<?> getEnclosingClass() {
+            return SampleBenchmark.class;
```

Comment:
Might as well `return this;` or does that not work?

Reply:
That doesn't work. Like I said above `io.jenkins.plugins.casc.SampleBenchmark` becomes `io.jenkins.plugins.casc.generated.SampleBenchmark_MyState_jmhType`